pkg/common/config/config.go

```diff
@@ -786,3 +787,23 @@ func GetSessionUserAgent(ctx context.Context) (string, error) {
 	}
 	return useragent, nil
 }
+
+// String returns a string representation of VirtualCenterConfig with sensitive fields redacted
+func (vc VirtualCenterConfig) String() string {
```

Comment:
Can we rename this method to something else ? May be RedactVCCreds() or similar ?

Reply:
we could, but then if someone does a `printf("%+v",vc)` somewhere else and doesn't use the `RedactedString()` function, then the password will be leaked there as well.

it would simplify the code as reflection wouldn't be needed, but I think it is safer to overload the `String` function.

what's your take on that ?

Reply:
That sounds good. We can overload the String function to keep things simpler. Thanks